bin/node-template/node/Cargo.toml

```diff
@@ -13,6 +13,9 @@ build = "build.rs"
 [package.metadata.docs.rs]
 targets = ["x86_64-unknown-linux-gnu"]
 
+[lib]
+name = "node_template"
+
```

Comment:
Why are you using the template and not the normal substrate node for tests?  
We should keep the template i pristine shape for external teams to fork.

Reply:
Good call! The normal substrate node already exports a lib, so it makes much more sense to use it.